Packs/Alexa/Integrations/Alexa/Alexa.py

```diff
@@ -106,11 +101,27 @@ def test_module_command():
 
 """EXECUTION BLOCK"""
 try:
+    params = demisto.params()
+    domain = demisto.args().get('domain')
```

Comment:
let's put this line in `demisto.command() == 'domain'` as we do not have domain when running the test module

Reply:
good catch, thanks!